controllers/OverviewController.php

```diff
@@ -45,4 +48,35 @@ public function actionShareholderList()
     {
         return $this->render('shareholder-list', ['asset' => AssetHelper::getSpaceAsset($this->contentContainer)]);
     }
+
+    public function actionPurchaseCoin()
+    {
+        $amountModel = new DynamicModel(['amount']);
```

Comment:
Why not creating a dedicated form ? maybe this DynamicModel will containg some other fields in upcoming updates ?

Reply:
Was thinking not worth doing a full Form for just one field, but I'll make one if more fields should come